### ISSUE TITLE
chip: default remove icon is different from 'close' icon

### ISSUE THREAD

**Issue by vdegenne** [material-components/material-web#5280]
### What is affected?

Component

### Description

when `<md-icon slot="remove-trailing-icon">close</md-icon>` is used in `filter-chip`  or `input-chip` to emulate the appearance of the default "remove" icon, the result is slightly different than the original.  

Here's a capture:

![289800641-32df580a-461a-4666-82ee-009c08cca402](https://github.com/material-components/material-web/assets/2827383/f75cf455-4a8c-437a-9851-0a40894cc686)

(top: default, bottom: using `close` icon)

In other words the default icon is bigger than material symbols.

### Reproduction

see capture above

### Workaround

undefined

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

v1.0.1

### Browser/OS/Node environment

everywhere
